user-guides/wallets/wallets-and-ocean.md

```diff
@@ -1 +1,35 @@
+---
+description: >-
+  How to use a crypto wallet to check your OCEAN balance and send OCEAN
```

Comment:
Now it is FET token, should we keep support for OCEAN? I think we should update with the tokens that are active and support transactions.